src/pymor/reductors/h2.py

```diff
@@ -644,6 +644,201 @@ def reconstruct(self, u):
+            - initial reduced-order model (|LTIModel|).
+
+            If the order of reduced model is given, initial
+            interpolation data is generated randomly.
+        tol
+            Tolerance for the convergence criterion.
+        maxit
+            Maximum number of iterations.
+        num_prev
+            Number of previous iterations to compare the current
+            iteration to. A larger number can avoid occasional cyclic
+            behavior.
+        conv_crit
+            Convergence criterion:
+
+            - `'sigma'`: relative change in interpolation points
+            - `'htwogap'`: :math:`\mathcal{H}_2-gap` distance of
+              reduced-order models divided by :math:`\mathcal{L}_2`
+              norm of new reduced-order model
+            - `'ltwo'`: relative :math:`\mathcal{L}_2` distance of
+              reduced-order models
+        projection
+            Projection method:
+
+            - `'orth'`: projection matrix is orthogonalized with respect
+              to the Euclidean inner product,
+            - `'Eorth'`: projection matrix is orthogonalized with
+              respect to the E product.
+
+        Returns
+        -------
+        rom
+            Reduced |LTIModel| model.
+        """
+        if not self.fom.cont_time:
+            raise NotImplementedError
+
+        self._clear_lists()
+        sigma, b, c = self._rom0_params_to_sigma_b_c(rom0_params)
+        self._store_sigma_b_c(sigma, b, c)
+        assert projection in ('orth', 'Eorth')
+
+        self.logger.info('Starting gap IRKA')
+        self._conv_data = (num_prev + 1) * [None]
+        if conv_crit == 'sigma':
+            self._conv_data[0] = sigma
+        for it in range(maxit):
+            self._set_V_W_reductor(sigma, b, c, projection)
+            rom = self._pg_reductor.reduce()
+            sigma, b, c, gap_rom = self._unstable_rom_to_sigma_b_c(rom)
+            self._store_sigma_b_c(sigma, b, c)
+            self._update_conv_data(sigma, gap_rom, rom, conv_crit)
+            self._compute_conv_crit(rom, conv_crit, it)
+            if self.conv_crit[-1] < tol:
+                break
+
+        return rom
+
+    def _rom0_params_to_sigma_b_c(self, rom0_params):
+        self.logger.info('Generating initial interpolation data')
+        self._check_rom0_params(rom0_params)
+        if isinstance(rom0_params, Integral):
+            sigma, b, c = self._order_to_sigma_b_c(rom0_params)
+        elif isinstance(rom0_params, np.ndarray):
+            sigma = rom0_params
+            _, b, c = self._order_to_sigma_b_c(len(rom0_params))
+        elif isinstance(rom0_params, dict):
+            sigma = rom0_params['sigma']
+            b = rom0_params['b']
+            c = rom0_params['c']
+        else:
+            sigma, b, c, _ = self._unstable_rom_to_sigma_b_c(rom0_params)
+        return sigma, b, c
+
+    def _set_V_W_reductor(self, sigma, b, c, projection):
```

Comment:
Maybe I'm missing something, but is there a reason for not using `LTIBHIReductor`?